buildroot-external/configs/generic_aarch64_defconfig

```diff
@@ -71,6 +71,9 @@ BR2_PACKAGE_LIBCGROUP=y
 BR2_PACKAGE_LIBCGROUP_TOOLS=y
 BR2_PACKAGE_BLUEZ5_UTILS=y
 BR2_PACKAGE_BLUEZ5_UTILS_CLIENT=y
+BR2_PACKAGE_BLUEZ5_UTILS_PLUGINS_AUDIO=y
+BR2_PACKAGE_BLUEZ5_UTILS_PLUGINS_HID=y
+BR2_PACKAGE_BLUEZ5_UTILS_PLUGINS_HOG
```

Comment:
To me this looks like you are missing the final `=y` at the end of the line. Isn't it?

Reply:
Right, thanks for spotting this!

```suggestion
BR2_PACKAGE_BLUEZ5_UTILS_PLUGINS_HOG=y
```

Reply:
FWIW, it seems that Kconfig/Buildroot automatically makes a `BR2_PACKAGE_BLUEZ5_UTILS_PLUGINS_HOG=y` out of it. That is why I didn't notice when building locally. Still, should be with `=y`.